test/e2e/app-dir/app-prefetch/prefetching.test.ts

```diff
@@ -232,7 +232,7 @@ describe('app dir - prefetching', () => {
     })
 
     const prefetchResponse = await response.text()
-    expect(prefetchResponse).toContain('Page Data!')
+    expect(prefetchResponse).not.toContain('Page Data!')
```

Comment:
This test was asserting on incorrect behavior. Since the fetch was initiated to a segment that was already being rendered, there'd be no new prefetch data to return.